frontend/resources/public/index.html

```diff
@@ -16,6 +16,12 @@
     <div id="toaster"></div>
     <script src="https://use.fontawesome.com/d25c01a5e8.js"></script>
     <script src="js/app.js"></script>
+    <script>
+      imas_seamap.db.api_url_base = "http://localhost:8000/api/";
+      imas_seamap.db.media_url_base = "http://localhost:8000/media/";
+      imas_seamap.db.wordpress_url_base = "http://localhost:8888/";
+      imas_seamap.db.img_url_base = "/img/";
+    </script>
```

Comment:
Huh, I hadn't thought of doing it that way!  I think it should be safe (the risk of name-mangling always makes me nervous, but you can protect against that.. actually have you tested with advanced optimisations on?)

The monkey-punching aspect of it I'm not a fan of though.  How about something (roughly) like:
```javascript
<script>
var
  api_url_base = "http://localhost:8000/api/",
  media_url_base = "....";
imas_seamap.core.init(api_url_base, media_url_base,...);
</script>
```
Ie, seamap is almost just a library; you pass it configuration params as it starts up?

Reply:
Fixed!